api/controller/notification.js

```diff
@@ -0,0 +1,36 @@
+import nodemailer from "nodemailer";
+
+class NotificationService {
+  constructor() {
+    this.transporter = nodemailer.createTransport({
+      service: "gmail",
+      auth: {
+        user: process.env.Email,
+        pass: process.env.Key,
+      },
+    });
+  }
+
+  async sendEmail({ email, subject, message }) {
+    const mailOptions = {
+      from: email,
+      to: "mustafizkaifee04+globehoppin@gmail.com",
+      subject: `Message From Globe Hoppin`,
```

Comment:
Subject will also change for different email. Pass subject as argument too.

Reply:
> Subject will also change for different email. Pass subject as argument too.

subject is already a field this is the title of the email

![image](https://github.com/user-attachments/assets/31b76bd0-adf5-4540-9e47-fe0cbf3aaefa)

Reply:
Yes, that subject is the actual subject of email, and the one in text is email body.
```suggestion
      subject,
```